src/renderer/component/fileViewer/internal/play-button.jsx

```diff
@@ -15,10 +15,19 @@ class VideoPlayButton extends React.PureComponent<Props> {
     const { fileInfo, mediaType, isLoading, play } = this.props;
     const disabled = isLoading || fileInfo === undefined;
     const doesPlayback = ['audio', 'video'].indexOf(mediaType) !== -1;
-    const icon = doesPlayback ? icons.PLAY : icons.EYE;
     const label = doesPlayback ? __('Play') : __('View');
 
-    return <Button button="primary" disabled={disabled} label={label} icon={icon} onClick={play} />;
+    return (
+      <Button
+        disabled={disabled}
+        label={label}
+        className={classnames('btn--icon', {
+          'btn--play': doesPlayback,
+          'btn--view': !doesPlayback,
```

Comment:
This should still use the `icon` prop.

`icon={ICONS.PLAY}`

Reply:
The icon I'm using here is not in the react-feathers repo.